qiskit/transpiler/preset_passmanagers/level0.py

```diff
@@ -162,7 +164,10 @@ def _direction_condition(property_set):
 
     _direction = [GateDirection(coupling_map)]
 
-    # 7. Unify all durations (either SI, or convert to dt if known)
+    # 7. Add calibrations for pulse gates
+    _calibrations = PulseGates(inst_map=inst_map)
```

Comment:
Should this object only be created if required? Eg., it should be contained within the block on line 203? The same in the other pass managers.

Reply:
Code is upgraded in https://github.com/Qiskit/qiskit-terra/pull/6759/commits/4bfc87b41ae210aec03b79e4233816543a50f44e